### DISCUSSION TITLE
一个很强的搜本站soutubot.moe

### DISCUSSION THREAD

**Discussion by Miuzarte** [Tsuk1ko/CQ-picfinder-robot#401] [一般讨论]
[]()
[]()

**Comment by Tsuk1ko** [comment]
别想了，它有cf五秒盾（

**Reply by Miuzarte** [reply]
草，我看到别人bot接入了还以为能用

**Reply by Tsuk1ko** [reply]
有吗，不会是跟作者py的吧

其实理论上是可以搞的，但是无头浏览器现在不好过 cf，得搞虚拟桌面，挺麻烦的

**Reply by Miuzarte** [reply]
> 有吗，不会是跟作者py的吧
> 
> 其实理论上是可以搞的，但是无头浏览器现在不好过 cf，得搞虚拟桌面，挺麻烦的

不太清楚  他好像是没用虚拟桌面
![image](https://user-images.githubusercontent.com/66856838/226087110-9b776b9e-d27a-4c47-a059-385763302e09.png)
![image](https://user-images.githubusercontent.com/66856838/226087191-ff7a7aad-e55a-45e8-b6f3-f92ff4efd221.png)
![image](https://user-images.githubusercontent.com/66856838/226087195-a45ba7d7-6d65-4903-b81a-e3588fc0ce40.png)
![O~4QQ9RCMATPD5L71D4)G}B](https://user-images.githubusercontent.com/66856838/226087198-85789bc8-4492-4cff-9456-b0e0cdbf7d53.png)

**Reply by Tsuk1ko** [reply]
🤔难道站和站也不同，他是 linux 么

我当时记得无头过不了 nhentai 的五秒盾，会卡住，搜了一堆 issue 有说有 stealth 对最新的 cf 不起作用了啥的

有空再看看吧
